articles/site-recovery/avs-tutorial-prepare-avs.md

```diff
@@ -20,9 +20,9 @@ This is the second tutorial in a series that shows you how to set up disaster re
 In this article, you learn how to:
 
 > [!div class="checklist"]
-> * Prepare an account on the vCenter server or vSphere ESXi host, to automate VM discovery.
-> * Prepare an account for automatic installation of the Mobility service on VMware VMs.
-> * Review VMware server and VM requirements and support.
+> * Prepare an account on the vCenter Server to automate VM discovery.
+> * Prepare an account for automatic installation of the Mobility service on VMware vSphere VMs.
```

Comment:
"VMware VMs" is used across our products and renaming it in just our documentation may cause confusion. So. let's not add 'vSphere'.

Reply:
@Sharmistha-Rai - unfortunately VMware VMs is a misnomer and we need to follow the naming guidelines of VMware. VMware vSphere is VMware product that runs Virtual Machines.